client/src/app/components/AppDialogShell.vue

```diff
@@ -91,7 +91,10 @@ export default {
       this.top += deltaY
     },
     onOpen () {
-      if (this.isModal) {
+      // console.log('onOpen >> ', this.$route, this)
+      // There seems to be a conflict when setting the body's position to fixed
+      // It conflicts and causes the bug mentioned here: https://github.com/BCcampus/edehr/issues/152
+      if (this.isModal && this.$route.meta.layout !== 'outside') {
```

Comment:
Please explain why it works in the ehr side but not the outside?  Probably because of the nesting of containers.

Reply:
I think so, as well. I compared the styles on it yesterday and I think that it is a question of a conflict between `display: flex` and `position:fixed`. I think that this happens in this case and not on the other due to the difference of inside having the sidebar/navbar and outside not having it. If needed, I can further investigate the issue and give you a more concrete answer... Thank you!